src/grovel.lisp

```diff
@@ -1,6 +1,6 @@
 (in-package :jpeg-turbo)
 (include "turbojpeg.h")
-#+freebsd
+#+(or freebsd OS-MACOSX)
```

Comment:
For macos (feature is `OS-MACOSX`), global include path also is `/usr/local/include`.